docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3"
+x-logging:
+  &default-logging
+  logging:
+    driver: json-file
+    options:
+      max-size: 100m
```

Comment:
Not working, please check this file.

Reply:
@krysal this is happening because I'm using the newer version of `docker-compose` (1.28.0). However, I have removed it and now default logging config is being used